### PR TITLE
Add arabic translation locale

### PR DESCRIPTION
**Pull request by MohamedHassanWD**

### CODE REVIEW

File: src/IntlProvider/locales/ar_EG.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+
+exports.__esModule = true;
+exports.default = void 0;
+
+var _extends2 = _interopRequireDefault(require("@babel/runtime/helpers/extends"));
+
+var Calendar = {
+  sunday: 'حد',
```
Comment (MohammadYounes) [inline src/IntlProvider/locales/ar_EG.ts]: @MohamedHassanWD I believe one letter abbreviation is much better.

sunday: 'ح',
monday: 'ن',
tuesday: 'ث',
wednesday:'ر',
thursday: 'خ',
friday: 'ج',
saturday: 'س'

Moreover, run the command `npm run format` to update files formatting (avoiding build failure).

File: src/IntlProvider/locales/ar_EG.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    inited: 'البدء',
+    progress: 'جاري الرفع',
+    error: 'خطأ',
+    complete: 'تم الإنتهاء',
+    emptyFile: 'فارغ',
+    upload: 'رفع'
+  }
+};
+exports.default = _default;
+module.exports = exports.default;
```
Comment (simonguo) [inline src/IntlProvider/locales/ar_EG.ts]: Thank you for adding Arabic.
You should change the file to ES Module.

Comment (Saleh56) [reply src/IntlProvider/locales/ar_EG.ts]: > Thank you for adding Arabic.
> 
> You should change the file to ES Module.

**Review by salemkmsr — COMMENTED** [review]
U

**Pull request merged**